### DISCUSSION TITLE
Support for firmware 1.12.x

### DISCUSSION THREAD

**Discussion by bvk007** [fabianishere/udm-iptv#98] [Q&A] [answered]
Hi Fabian (and others),

Has anyone already tested with the new EA firmware versions 1.12.x and network 7.1.x ?

Cheers :)

**Comment by fabianishere** [comment, marked as answer]
I haven't, but I see no reason why it wouldn't work with the new firmware, seeing the changelog.

**Reply by bvk007** [reply]
Right...I'll give it a go then later on...keep you posted. Thnx :)

**Reply by bvk007** [reply]
Yep all is good!!
